pkg/kubelet/kubelet.go

```diff
@@ -339,6 +399,32 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) setupTmpfsDataDirs() error {
+	kl.tmpfsRootDirectory = path.Clean(kl.tmpfsRootDirectory)
+	// TODO: handle edge cases like mount already exists
```

Comment:
Yeah this TODO is a big one.  We have an "isMountPoint" function floating about somewhere.  Maybe a good proxy.  Or actually process /proc/mounts (we have a function for that too in mount.Interface)